docs/fsharp/language-reference/type-abbreviations.md

```diff
@@ -10,12 +10,14 @@ A *type abbreviation* is an alias or alternate name for a type.
 ## Syntax
 
 ```fsharp
-type type-abbreviation = type-name
+type [accessibility-modifier] type-abbreviation = type-name
 ```
 
 ## Remarks
 You can use type abbreviations to give a type a more meaningful name, in order to make code easier to read. You can also use them to create an easy to use name for a type that is otherwise cumbersome to write out. Additionally, you can use type abbreviations to make it easier to change an underlying type without changing all the code that uses the type. The following is a simple type abbreviation.
 
+Accessibility of type abbreviations defaults to `public`.
```

Comment:
should you link to https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/access-control for more info?

Reply:
I think it's okay for now, since this is searchable in the ToC. I'm thinking of doing a pass-over of various docs and linking more deeply, but I haven't established principles for how to do that yet.